### ISSUE TITLE
Feature request: fit time holes

### ISSUE THREAD

**Issue by emc369** [kimai/kimai#718]
Thanks for continuing this software!

Similar approach to [#411](https://github.com/kimai/kimai/issues/411)
But use + and - to fit the holes in between or extend with just one mouse click the time.

When I'm at a customer usually my timesheet looks like in the screenshot. Problem is, I can't always start or stop the timer. Nor open a new one. But afterwards I need to extend the entries.
So it would be nice that I have + and - feature to extend "In" and "Out" time. 
![easy_fit_time_holes](https://cloud.githubusercontent.com/assets/20052110/16208507/cd54aa78-3732-11e6-8b29-129fa57c0db4.png)

1. I round to 15 min so for me it would be good to use that as increment.
2. Or instead to extend till the next time entry.

For the Date it could be implemented too.

**Comment by simonschaufi** [comment]
Did I understand you right that you just want to update the time or do you want to add new entries also in between? What exactly whould happen when I click the plus or minus button? Just increase or decrease the time for 1min/15min?

**Comment by emc369** [comment]
Yes you understood it right. I would like to increase or decrease time by 15 min when I press the plus or minus button. 

I could sponsor a small amount. How would that work? 
(Github doesn't provide pm? Simon könntest du mir dies erklären -> pug@gmx.ch Danke!)

**Comment by simonschaufi** [comment]
I'm just brainstorming about a good UI since the plus and minus button don't fit into the UI.

What comes to my mind is a new layer where you can drag and drop your times how you want like stacked bars:
* http://www.highcharts.com/demo/column-stacked

with something like this combined:
* https://jqueryui.com/slider/#rangemax
* http://www.actioniq.co/react-resizable-bar/

As described in the Readme, you have multiple options. The sponsoring with the fewest fees is of course SEPA transaction but you can also use bountysource (The developer who completes the task gets the money) or paypal of course.

PS: Yeah sadly Github does not provide a direct messaging (yet). I hope they do at some point.

**Comment by emc369** [comment]
I like your ideas. With that it would be easier to add/edit entries on a mobile too.

And if you and the other developers are brainstorming for a new UI, it doesn't make sense to implement this feature. Please remove the bountysource and Feature Request.

It will be interesting to see what else you have in mind for a new UI.
